ajax/libs/cssobj/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cssobj",
+  "description": "Runtime CSS manager, Turn CSS into dynamic JS module, Stylesheet CRUD (Create, Read, Update, Delete) in CSSOM, Solve common problems of CSS-in-JS, name space (local) class names, media query hook for IE8, styled components",
```

Comment:
Let's remove ", media query hook for IE8, styled components" part as @ futurist said (will be also the same as the upstream).
@futurist Thanks again! :smile: